woocommerce/woocommerce-ios
# Replaces About screen with unified Automattic About screen

### Description

This PR introduces the new [unified Automattic About screen](https://github.com/Automattic/AutomatticAbout-Swift) to the app, replacing the existing About screen. This contains links to social accounts, websites, the ability to rate and share the app, and links to legal policies and third party licenses.

I noticed that the original About screen included a crash debugging gesture (tapping 4 times on the app icon). The new About screen framework currently doesn't support adding functionality like this directly to the header view, so for now I've moved this debugging gesture to the heart icon in the bottom of the Settings screen. If this isn't acceptable, I can update the AutomatticAbout framework to allow a custom gesture to be added to the header view.

The other known issue is that I ran out of time to add analytics events to the various rows within the About screen. However, I don't think the previous screen had this either, so this isn't a regression and can be added as a future enhancement.
 
### Testing instructions

* Build and run the app and navigate to Menu > Settings > WooCommerce (under 'About the App').
* Interact with each item in the screen and ensure it performs as you'd expect. The "Share with a Friend" item may be easier to test on a device, as you're very limited for sharing options on the Simulator.
* On the Settings screen, try tapping the heart icon 4 times quickly to test the crash debug gesture.

### Screenshots

|   |   |
|---|---|
| ![Simulator Screen Shot - iPhone 13 Pro - 2022-03-11 at 22 43 14](https://user-images.githubusercontent.com/4780/157985014-99ef3463-e80b-402e-8c77-4a33de163312.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2022-03-11 at 22 52 42](https://user-images.githubusercontent.com/4780/157985161-635a9f9b-bab5-4503-bb85-7dba0ee96135.png) |

---
